OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameRadarDisplayLogic.cs

```diff
@@ -25,11 +26,12 @@ public IngameRadarDisplayLogic(Widget widget, World world)
 			var blockColor = Color.Transparent;
 			var radar = widget.Get<RadarWidget>("RADAR_MINIMAP");
 			radar.IsEnabled = () => radarEnabled;
+			var devMode = world.LocalPlayer.PlayerActor.Trait<DeveloperMode>();
```

Comment:
You are now requiring `DeveloperMode`. Either `Requires<>` it or do a null check.